package.json

```diff
@@ -59,6 +59,7 @@
     "chai-spies": "^1.x",
```

Comment:
Missing lock file update, are you using old version for `npm`?

Reply:
ah sorry, I'm in the habit of using yarn